09-coap/test_spec09.py

```diff
@@ -12,6 +12,7 @@
 
 from riotctrl.shell import ShellInteractionParser
```

Comment:
This can be removed, I think. I don't find any mention of it in the code now. I am a bit worried though, that it is not picked up by the linters.

Reply:
I ran all the test in `test_spec09.py` with:
```bash
sudo RIOTBASE=$(readlink -f ../RIOT) tox -e test -- -k "spec09" --non-RC
```
All passes so maybe its okay to remove it.